ISSUE_TRIAGE_PROCESS.rst

```diff
@@ -27,16 +24,116 @@ of resolving issues.
+more ways to help others.
+
+The triage team members do not have committer privileges but they can
+assign, edit, and close issues and pull requests without having capabilities to merge the code. They can
+also convert issues into discussions and back. The expectation for the issue triage team is that they
+spend a bit of their time on those efforts. Triaging means not only assigning the labels but often responding
+to the issues and answering user concerns or if additional input is needed - tagging the committers or other community members who might be able to help provide more complete answers.
+
+Being an active and helpful member of the "Issue Triage Team" is actually one of the paths towards
+becoming a committer. By actively helping the users and triaging the issues and responding to them and
+involving others (when needed) shows that you are not only willing to help our users and the community,
+but are also ready to learn about parts of the projects you are not actively contributing to - all of that
+are super valuable components of being eligible to `become a committer <COMMITTERS.md>`_.
+
+If you are a member of the triage team and not able to make any commitment, it's best to ask to have yourself
+removed from the triage team.
+
+BTW. Every committer is pretty much automatically part of the "Issue Triage Team" - so if you are committer,
+feel free to follow the process for every issue you stumble upon.
+
+Actions that can be taken by the issue triager
+''''''''''''''''''''''''''''''''''''''''''''''
+
+There are several actions an issue triager might take:
+
+* Closing and issue with "invalid" label explaining why it is closed in case the issue is invalid. This
+  should be accompanied by information that we cn always re-open an issue if our understanding was wrong
+  or if the user provides more information.
+
+* Converting an issue to a discussion, if it is not very likely it is an Airflow issue or when it is not
+  responsible, or when it is a bigger feature proposal requiring discussion or when it's really users
+  troubleshooting or when the issue description is not at all clear. This also involves inviting the user
+  to a discussion if more information might change it.
+
+* If the issue seems important enough that it should likely be looked at before the next release but there
+  is not enough information or doubts on why and what can be fixed, assign the issue to the milestone of
+  the next bugfix - then, no matter what the issue will be looked at by the release manager and it might
+  trigger additional actions during the release preparation. This is usually followed by one of the next
+  actions.
+
+* Actually fixing the issue in a PR if you see it is easy to fix. This is a great way also to learn and
+  contribute to parts that you usually are not contributing to, and sometimes it is surprisingly easy.
+
+* Assign "good first issue" label if an issue is clear but not important to be fixed immediately, This
```

Comment:
```suggestion
* Assigning "good first issue" label if an issue is clear but not important to be fixed immediately, This
```

To be consistant with other actions (asking, fixing, calling, assigning, converting...)